packages/server/test/interop/react/useQuery.test.tsx

```diff
@@ -6,6 +6,8 @@ import { render, waitFor } from '@testing-library/react';
 import { expectTypeOf } from 'expect-type';
 import React, { useState } from 'react';
 
+jest.retryTimes(3);
```

Comment:
Did you mean to add these?

Reply:
I think it sometimes get flaky in Github Actions

Reply:
Yes. I did intend to. I can't be arsed doing a proper fix 😬

Reply:
Ah ok, we can look at them later